Jenkinsfile

```diff
@@ -99,6 +99,8 @@ def runTests(dockerVersion, pythonVersion, baseImage) {
                       --privileged \\
                       --volume="\$(pwd)/.git:/code/.git" \\
                       --volume="/var/run/docker.sock:/var/run/docker.sock" \\
+                      --volume="\$(pwd)/.docker/config.json:/code/.docker/config.json" \\
```

Comment:
I think it needs to be in the home directory, so I assume it is `/root/.docker/config.json` in the container. On the host probably in $HOME.

Reply:
Looks like `script/test/all ` uses `~/.docker/ ...`, which should also work

Reply:
(for the "host" side that is 😅)